src/common/vitals/cumulative-layout-shift.js

```diff
@@ -1,13 +1,19 @@
-import { onCLS } from 'web-vitals'
+import { onCLS } from 'web-vitals/attribution'
 import { VITAL_NAMES } from './constants'
 import { VitalMetric } from './vital-metric'
 import { isBrowserScope } from '../constants/runtime'
 
 export const cumulativeLayoutShift = new VitalMetric(VITAL_NAMES.CUMULATIVE_LAYOUT_SHIFT, (x) => x)
 
 if (isBrowserScope) {
-  onCLS(({ value, entries }) => {
-    if (cumulativeLayoutShift.roundingMethod(value) === cumulativeLayoutShift.current.value) return
-    cumulativeLayoutShift.update({ value, entries })
+  onCLS(({ value, attribution, id }) => {
+    const attrs = {
+      metricId: id,
```

Comment:
Also curious what the `metricId` is/will be used for here

Reply:
Similar to metricId for INP in that if you want the final CLS for a view, you have to query/filter the new cls nodes sent by `latest(metricId)` to get that value. This is the most accurate by-the-books approach